tests/power/devicepm/test_suspendresume_deepsleep/src/test_suspendresume_deepsleep.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ztest.h>
+
+/*macros*/
```

Comment:
I think we can remove these comments. /* macros*/, /*external variables*/ and /*local variables*/.

Reply:
Done